test/test-applications/regression/StackExchange.Redis.AssemblyConflict.LegacyProject/Program.cs

```diff
@@ -17,16 +17,16 @@ public static async Task Main()
             }
             finally
             {
-                var instrumentationType = Type.GetType("Datadog.Trace.ClrProfiler.Instrumentation, Datadog.Trace.ClrProfiler.Managed");
+                var instrumentationType = Type.GetType("Datadog.Trace.ClrProfiler.Instrumentation, OpenTelemetry.Instrumentation.ClrProfiler.Managed");
                 var profilerAttached = instrumentationType?.GetProperty("ProfilerAttached", BindingFlags.Public | BindingFlags.Static)?.GetValue(null) ?? false;
-                var tracerAssemblyLocation = Type.GetType("Datadog.Trace.Tracer, Datadog.Trace")?.Assembly.Location ?? "(none)";
+                var tracerAssemblyLocation = Type.GetType("Datadog.Trace.Tracer, OpenTelemetry.Instrumentation")?.Assembly.Location ?? "(none)";
```

Comment:
I think that this change is wrong. The name of the assembly has changed. Not the type's FullName.

Reply:
As far as I understand the first is Class name, the second assembly name.

Reply:
You are right